src/index.scss

```diff
@@ -14,7 +14,7 @@ body {
   overflow-y: overlay;
   display: flex;
   flex-direction: column;
-  font-family: var(--font-default);
+  font-family: HelsinkiGrotesk-Regular, Arial, sans-serif;
```

Comment:
Nice catch 👍 
Could this be in the variables as app-body-text or something?

Reply:
- [ ] Put body font-family to its own variable 'app-body-text'